### PR TITLE
Raspberry Pi NO_AES

### PR DESCRIPTION
**Pull request by ArqTras**
Fixed #7946

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -464,7 +464,7 @@ application.
 
 ### On FreeBSD:
 
-The project can be built from scratch by following instructions for Linux above(but use `gmake` instead of `make`).
```
Comment (mj-xmr) [inline README.md:467]: There's a rule here, that such changes (removing whitespaces in a line, where you don't change the merit)  aren't welcome in a PR.

**Comment by mj-xmr** [comment]
I can start a PR and make the changes, that you proposed in one commit, if you have nothing against it. You will be set as the PR's author of course.

**Comment by ArqTras** [comment]
Ok

**Comment by selsta** [comment]
#7953

**Pull request closed without merging**